src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Ball {
+
+    private final String color;
+    private final int number;
```

Comment:
I like this nice touch of making those fields final. Makes total sense here, GJ! :+1: 
Just be careful with that in the future, because usually, our POJO classes like this are mutable
You can read more about it [here](https://www.baeldung.com/java-immutable-object)